bigtable/examples/bigtable_samples.cc

```diff
@@ -238,7 +260,7 @@ void ReadModifyWrite(bigtable::Table& table) {
 
 //! [sample row keys]
 void SampleRows(bigtable::Table& table) {
-  auto samples = table.SampleRows<std::vector>();
```

Comment:
Ditto here: was this intended as part of this PR?

Reply:
Yes, I wanted a simple example with the default collection type, and a separate one with other collection types.